instrumentation/github.com/gin-gonic/gin/otelgin/option.go

```diff
@@ -22,8 +22,10 @@ import (
 )
 
 type config struct {
-	TracerProvider oteltrace.TracerProvider
-	Propagators    propagation.TextMapPropagator
+	TracerProvider   oteltrace.TracerProvider
+	Propagators      propagation.TextMapPropagator
+	SpanStartOptions []oteltrace.SpanStartOption
```

Comment:
I suggest adjusting this to private. `spanStartOptions`.

`SpanEndOptions`  is the same.